docs/tutorialkit.dev/src/content/docs/reference/react-components.mdx

```diff
@@ -17,6 +18,8 @@ import sourceExampleSimpleEditor from '@components/react-examples/ExampleSimpleE
 import sourceExampleFileTree from '@components/react-examples/ExampleFileTree?raw';
 import sourceExampleCodeMirrorEditor from '@components/react-examples/ExampleCodeMirrorEditor?raw';
 import sourceExampleTerminal from '@components/react-examples/ExampleTerminal?raw';
+import sourceUseTheme from "@components/react-examples/hooks/useTheme?raw";
```

Comment:
I would use a different version from that file. That file is very specific to our docs.

Here's what we could show for `hooks/useTheme.ts`:

```ts
export function useTheme() {
  return 'light';
}
```

Reply:
We could also remove the hook usage completely and just use static `"light"` string as `theme` in the source code. What do you think?

Reply:
Hmm but then the docs wouldn't look as good when using viewing them in dark theme.

What drawback do you see in showing a different version for `useTheme`? I don't expect that hook to be changed very often.

Reply:
Ah no worries! :raised_hands: